clouddq/classes/dq_entity.py

```diff
@@ -362,6 +364,38 @@ def from_dataplex_entity(
                 f"is unsupported for entity:\n {dataplex_entity.to_dict()}"
             )
 
+    @classmethod
+    def from_bq_entity_uri(
+        self, entity_uri: EntityUri, bigquery_client: BigQueryClient
+    ) -> DqEntity:
+        project_id = entity_uri.get_configs("projects")
+        table_name = entity_uri.get_table_name()
+        configs = entity_uri.configs_dict
+        entity_id = entity_uri.get_entity_id()
+        columns_dict = bigquery_client.get_table_schema(
+            table=table_name, project_id=project_id
+        )
+        entity_configs = {
+            "source_database": "BIGQUERY",
+            "resource_type": "BIGQUERY",
+            "table_name": configs.get("tables"),
+            "dataset_name": configs.get("datasets"),
+            "project_name": configs.get("projects"),
+            "columns": columns_dict.get("columns"),
+            "environment_override": {},
+            "entity_id": entity_id,
+            "dataplex_name": configs.get("dataplex_name", None),
+            "dataplex_lake": configs.get("dataplex_lake", None),
+            "dataplex_zone": configs.get("dataplex_zone", None),
+            "dataplex_location": configs.get("dataplex_location", None),
+            "dataplex_asset_id": configs.get("dataplex_asset_id", None),
+            "dataplex_createTime": configs.get("dataplex_createTime", None),
+            "dataplex_updateTime": configs.get("dataplex_updateTime", None),
```

Comment:
If the code got to this point, it is safe to say there are no corresponding Dataplex entity_id for the entity_uri, therefore all "dataplex_*" fields should be set to 'None' to ensure correctness.

Reply:
Yes I agree. All this should be None by default if the code got to this point.